api/Performance.json

```diff
@@ -51,12 +51,26 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/clearMarks",
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
-            "chrome_android": {
-              "version_added": "46"
-            },
+            "chrome": [
+              {
+                "version_added": "29"
```

Comment:
Based on your PR description, I'd expect this to be version 28. Which number is correct?

(This goes for this entry, the version_removed below, and the equivalents in chrome_android in this feature, and their equivalents in `clearMeasures`)

Reply:
I double-checked. This should be 29. (I've updated the PR description.)